packer/server-certs.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#Required
+domain=$1
+vault_url=$2
+vault_token=$3
+commonname=$domain
+
+echo
+echo "Generating key request for $domain"
+echo
+
+password=dummypassword
+
+#Generate a key
+openssl genrsa -des3 -passout pass:$password -out $domain.key 2048 -noout
```

Comment:
These are client certs so that consul can authenticate to the https://localhost.../swarm endpoint, so I had thought that it would be pretty much the same thing that we do for people to authenticate to kibana, where the client generated their own key and had vault sign it.

Reply:
OK, either will work.